services/QuillLMS/app/controllers/cms/blog_posts_controller.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 class Cms::BlogPostsController < Cms::CmsController
+  around_action :force_writer_db_role, only: [:destroy, :unpublish]
```

Comment:
I think this audit will be helpful in flagging actions that use the wrong http verb like these. destroy as a `'GET'` can be dangerous